appinfo/info.xml

```diff
@@ -22,7 +22,7 @@ Guests users can only access files shared to them and cannot create any files ou
 	<screenshot>https://raw.githubusercontent.com/nextcloud/guests/master/screenshots/settings.png</screenshot>
 	<screenshot>https://raw.githubusercontent.com/nextcloud/guests/master/screenshots/dropdown.png</screenshot>
 	<dependencies>
-		<nextcloud min-version="21" max-version="23" />
```

Comment:
Not even 22 ?

Reply:
good point... let me double check if we updated the router there

Reply:
22 has router v1: https://github.com/nextcloud/server/blob/stable22/package.json#L41

Reply:
But why would another deps matter here?